Document new issue details event dropdown and the recommended event (#7652)
getsentry/sentry-docs: src/docs/product/issues/issue-details/index.mdx

Learn more about what's included an issue detail for each of these in [Error Issues](/product/issues/issue-details/error-issues/) and [Performance Issues](/product/issues/issue-details/performance-issues/).
In addition, this page provides several other key pieces of information.

## Event Dropdown

> 
> By default, the issues page will display the "Recommended" event. To view other events, you can used the dropdown to skip to the latest or oldest event, or view the full list of events. If you wish to change the default event, you may do so in your [User Settings](https://sentry.io/settings/account/details/#defaultIssueEvent).

Reviewer: ```suggestion

```

Merged this into the next paragraph and moved the view/change stuff to after. I think this way, it's a bit shorter to read and flows better with all the recommended event info in the same place.

> What is the "Recommended" event? The recommended event is the event that gives you the greatest context in solving the issue. Sentry uses the following criteria in determining the recommended event:

Reviewer: ```suggestion
By default, the **Issue Details** page displays the "Recommended" event with the most context to help you solve the issue. Sentry uses the following criteria in determining the recommended event:
```

> - **Recency**: The recommended event is never more than 7 days older than the lastest event.

Reviewer: ```suggestion
- **Recency**: The recommended event is never more than 7 days older than the latest event.
```

- **Relevence**: The recommended event takes into account what terms you've searched for on your way to the issue.
- **Content**: The recommended event prioritizes events that contain debugging tools such as replays, profiles, and traces.

> 

Reviewer: ```suggestion

To view other events, you can use the dropdown to skip to the latest or oldest event, or view the full list of events. You can change the default event in your [User Settings](https://sentry.io/settings/account/details/#defaultIssueEvent).
 
```
